sdks/go/alpha.go

```diff
@@ -172,3 +173,88 @@ func (a *Alpha) SetCounterCapacity(key string, amount int64) (bool, error) {
 	}
 	return true, err
 }
+
+// GetListCapacity returns the Capacity for a List, given the List's key (name).
+// Will error if the key was not predefined in the GameServer resource on creation.
+func (a *Alpha) GetListCapacity(key string) (int64, error) {
+	list, err := a.client.GetList(context.Background(), &alpha.GetListRequest{Name: key})
+	if err != nil {
+		return -1, errors.Wrapf(err, "could not get List %s", key)
+	}
+	return list.Capacity, nil
+}
+
+// SetListCapacity sets the capacity for a given list. Capacity must be between 0 and 1000.
+// Will error if the key was not predefined in the GameServer resource on creation.
+func (a *Alpha) SetListCapacity(key string, amount int64) (bool, error) {
+	_, err := a.client.UpdateList(context.Background(), &alpha.UpdateListRequest{
+		List: &alpha.List{
+			Name:     key,
+			Capacity: amount,
+		},
+		UpdateMask: &fieldmaskpb.FieldMask{Paths: []string{"capacity"}},
+	})
+	if err != nil {
+		return false, errors.Wrapf(err, "could not set List %s capacity to amount %d", key, amount)
+	}
+	return true, err
+}
+
+// ListContains returns if a string exists in a List's values list, given the List's key (name)
+// and the string value. Search is case-sensitive.
+// Will error if the key was not predefined in the GameServer resource on creation.
+func (a *Alpha) ListContains(key, value string) (bool, error) {
+	list, err := a.client.GetList(context.Background(), &alpha.GetListRequest{Name: key})
+	if err != nil {
+		return false, errors.Wrapf(err, "could not get List %s", key)
+	}
+	// TODO: Should I move this logic to the SDK Server, or is it OK here?
+	for _, val := range list.Values {
```

Comment:
Should I move this logic to the SDK Server, or is it OK here? Alpha proto & SDK have Add / Remove / Update / Get, but no "contains".